Samples/Samples/ViewModel/HomeViewModel.cs

```diff
@@ -204,6 +204,12 @@ public HomeViewModel()
                     typeof(VibrationPage),
                     "Quickly and easily make the device vibrate.",
                     new[] { "vibration", "vibrate", "hardware", "device" }),
+                new SampleItem(
+                    "📳",
+                    "HapticFeedback",
+                    typeof(HapticFeedbackPage),
+                    "HapticFeedback",
```

Comment:
The description from the page.

```suggestion
                    "Quickly and easily make the device provide haptic feedback",
```